src/main/kotlin/app/revanced/patches/youtubevanced/ad/general/annotations/YouTubeVancedCompatibility.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.youtubevanced.ad.general.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+    [
+        Package("com.vanced.android.youtube")
+    ]
+)
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+internal annotation class YouTubeVancedCompatibility
```

Comment:
Please rename the annotation to `GeneralAdsCompatibility` as our convention goes.
```suggestion
@Compatibility([Package("com.vanced.android.youtube")])
@Target(AnnotationTarget.CLASS)
internal annotation class YouTubeVancedCompatibility
```